### DISCUSSION TITLE
Intermittent error accessing Directus

### DISCUSSION THREAD

**Discussion by thepurpleblob** [axios/axios#4660] [Q&A]
Some of my users are reporting the "Oops" page and I cannot reproduce the problem. I've added some debugging and done some digging. It seems to be related to the Axios call to a Directus CMS. In the Promise, I am catching the error and stringifying the error object, this is the result...

![Screenshot_20220501-190228_Edge](https://user-images.githubusercontent.com/664139/166161056-1335feec-18c3-4cf6-878f-c3ebbf715b4d.jpg)

No error message, no nothing. Why is this failing (or what can I check next)? I'm currently clueless.

**Comment by jasonsaayman** [comment]
I cannot help you with this info, please can you create an issue with as much info as possible?
